src/header/launchbar/BlastLaunchbarButton.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { useAppSelector } from 'src/store';
+
+import { getUnviewedBlastSubmissions } from 'src/content/app/tools/blast/state/blast-results/blastResultsSelectors';
+
+import LaunchbarButtonWithNotification from './LaunchbarIconWithNotification';
+import { BlastIcon } from 'src/shared/components/app-icon';
+
+import { JobStatus } from 'src/content/app/tools/blast/state/blast-results/blastResultsSlice';
+
+import styles from './Launchbar.scss';
+
+const BlastLaunchbarButton = () => {
+  const unviewedSubmissions = useAppSelector(getUnviewedBlastSubmissions);
+  let isAnyJobRunning = false;
```

Comment:
why are we declaring it as a `let` here instead of declaring it as a const on line 34?

Reply:
Ah I failed to check this one after getting rid of a `useEffect` that I had for checking jobs that are running. Thanks for spotting this.

Reply:
✅